OpenRA.Mods.Common/Widgets/ConfirmationDialogs.cs

```diff
@@ -37,6 +48,22 @@ public static void PromptConfirmAction(string title, string text, Action onConfi
 				if (onCancel != null)
 					onCancel();
 			};
+
+			var restartButton = prompt.Get<ButtonWidget>("RESTART_BUTTON");
```

Comment:
Please use `GetOrDefault` followed by a null check.
Mods should not be required to add this button.